pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -157,7 +160,13 @@ public SubscribeRate getSubscribeRate() {
         return this.topicPolicies.getSubscribeRate().get();
     }
 
-    public DispatchRateImpl getSubscriptionDispatchRate() {
+    public DispatchRateImpl getSubscriptionDispatchRate(String subscriptionName) {
+        DispatchRateImpl rate = Optional.ofNullable(subscriptionPolicies.get(subscriptionName))
+                .map(SubscriptionPolicies::getDispatchRate).orElse(null);
+        rate = DispatchRateImpl.normalize(rate);
```

Comment:
Whether it should be judged here that ```rate``` is not empty

Reply:
`DispatchRateImpl` don't have concept of `empty`. `normalize` can handle the case when rate is `null`.